README.md

```diff
@@ -14,6 +14,48 @@ mode.
 * leveldb
 
 
+## Installing
+
+You will need a bitcoin node running on the same machine. You can safely use 
+the testnet. You will need this in your bitcoin.conf:
+```
+#testnet=1 # uncomment this for testnet
+txindex=1
+server=1
+listen=1
+rpcuser=CHANGETHIS
+rpcpassword=CHANGETHIS
+rpcserialversion=0 # ots doesn't support segwit yet
+```
+
+Clone the repo and install the requirements:
+
+```
+git clone https://github.com/opentimestamps/opentimestamps-server.git
+pip3 install -r requirements.txt
+```
+
+You need to define the public URI of your calendar server in ~/.otsd/calendar/uri .
+Basically whatever is in that file is put into the URI field of pending attestations returned by that calendar server. So when a client goes to verify a timestamp created by that calendar, if the URI starts with http or https, it'll try to make a HTTP(S) connection to that server to fetch the rest of the timestamp proof.
+
+E.g. the alice.btc.calendar.opentimestamps.org calendar server has https://alice.btc.calendar.opentimestamps.org in the ~/.otsd/calendar/uri file.
+
+You will also need a hmac key:
+``` 
+openssl rand -base64 128 > .otsd/calendar/hmac-key
```

Comment:
Probably better to use /dev/urandom, and also, the hmac-key is used as raw binary, not hex. So change that to

    dd if=/dev/random of=~/.otsd/calendar/hmac-key bs=32 count=1